plugins/powershell/Dockerfile

```diff
@@ -42,11 +42,16 @@ ADD . /python/src
 
 WORKDIR /python/src
 
-# End package dependencies
+ADD ./plugin.spec.yaml /plugin.spec.yaml
+ADD ./requirements.txt /python/src/requirements.txt
+
 RUN if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+
+ADD . /python/src
+
 RUN python setup.py build && python setup.py install
 
 # User to run plugin code. The two supported users are: root, nobody
-USER root
+USER nobody
```

Comment:
I think this was changed to `root` a long time ago to fix a customer bug at the time